bot/constants.py

```diff
@@ -7,17 +7,17 @@
 ######################################################################################################################################################
 """
 
-DEBUG = bool(os.environ['DEBUG'] == 'True') if 'DEBUG' in os.environ else False
+DEBUG = bool(os.environ.get('DEBUG') == "True")
 TELEGRAM_BOT_TOKEN = os.environ['TELEGRAM_BOT_TOKEN']
-SLACK_WEBHOOK = os.environ['SLACK_WEBHOOK'] if 'SLACK_WEBHOOK' in os.environ and os.environ['SLACK_WEBHOOK'] else None
+SLACK_WEBHOOK = os.environ.get('SLACK_WEBHOOK')
+
 
 # Set NODE_IP depending on mode (if None, certain node health jobs are not executed)
 if DEBUG:
     NODE_IP = 'localhost'
-elif 'NODE_IP' in os.environ and os.environ['NODE_IP']:
-    NODE_IP = os.environ['NODE_IP']
 else:
-    NODE_IP = None
+    NODE_IP = os.environ.get('NODE_IP') if os.environ.get('NODE_IP') else None
```

Comment:
```suggestion
    NODE_IP = os.environ.get('NODE_IP')
```

Reply:
No. 
Because if `NODE_IP` exists but is empty (`NODE_IP=`), then the NODE_IP isn't `None` but an empty string which produces errors.

Reply:
That's right, didn't notice that. But does it make a difference when `if NODE_IP:` still produces false?